lib/commentOnClosedIssue.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+var Cesium = require('cesium');
+var Promise = require('bluebird');
+var rp = require('request-promise');
```

Comment:
I'm sure their doc did it like this, but our convention is to use `requestPromise` as the variable name.